src/main/java/com/github/manolo8/darkbot/utils/StartupParams.java

```diff
@@ -135,4 +124,38 @@ public String toString() {
                 .map(e -> e.getKey().toString() + "= " + e.getValue().toString())
                 .collect(Collectors.joining(","));
     }
+
+    public static class AutoLoginProps {
```

Comment:
I'd keep this an internal class, not public and then keep the getAutoLoginValue, and other methods accessible directly via StartupParams
```suggestion
    private static class AutoLoginProps {
```

but again, this is mostly personal preference

Reply:
![image](https://user-images.githubusercontent.com/52008918/185479943-fbe951e0-9172-4a95-8f0b-71e94b4c0fa9.png)